Bech32 test fails (when using go 1.15?)

## Environment

* OS: Arch Linux x86_64, go 1.15
* age version: 1.0.0.beta4.r2.g21a7203-1

## What were you trying to do

Install age/age-git via the aur

## What happened

Installation fails because a test is failing

```
# filippo.io/age/internal/bech32_test
internal/bech32/bech32_test.go:46:12: conversion from untyped int to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
```
The problematic line looks like this: 
```
{"spl" + string(127) + "t1checkupstagehandshakeupstreamerranterredcaperred2y9e3w", false},
```
I guess `string(127)` needs to be `string(rune(127))`? The test passes with this change.

--- a/internal/bech32/bech32_test.go
+++ b/internal/bech32/bech32_test.go
@@ -43,7 +43,7 @@ func TestBech32(t *testing.T) {
 		{"split1a2y9w", false},      // too short data part
 		{"1checkupstagehandshakeupstreamerranterredcaperred2y9e3w", false}, // empty hrp
 		// invalid character (DEL) in hrp
-		{"spl" + string(127) + "t1checkupstagehandshakeupstreamerranterredcaperred2y9e3w", false},
+		{"spl" + string(rune(127)) + "t1checkupstagehandshakeupstreamerranterredcaperred2y9e3w", false},
 		// too long
 		{"11qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqsqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqc8247j", false},
 

